src/ui/fragments/list/SettingsFragmentForm.js

```diff
@@ -17,10 +17,27 @@ export class SettingsFragmentFormBody extends React.Component {
   }
 
   renderBody() {
-    return !this.props.error ?
+    const { alert } = this.props;
+    let alertMessage;
+    if (alert && alert === 'success') {
```

Comment:
There's no need to check if alert is truthy ('success' is truthy, so the check is redundant), `if (alert === 'success')` is enough